master/usage/decommissioning-a-node.md

```diff
@@ -2,14 +2,21 @@
 title: Decommissioning a Node
 ---
 
-### Why you might be interested in this guide
+### About decommissioning nodes
 
-- You are decomissioning a host running calico/node or removing it from your
+If you are running the [the kube-controllers node controller]({{site.baseurl}}/{{page.version}}/reference/kube-controllers/configuration)
```

Comment:
We should make sure we like this term and then  add it to the terminology page.

Reply:
Potentially "kube-controllers with the node controller enabled"

Reply:
Or just "running the node controller"

Reply:
It is a good point @ozdanborne ....I think `kube-controller` is the name of the image and we have actually standardized on "node controller" as the plain English equivalent, so I like Casey's second suggestion.